drivers/gpio/gpio_nrf5.c

```diff
@@ -66,7 +59,9 @@ struct _gpiote {
 };
 
 /*@todo: move GPIOTE channel management to a separate module */
-static u32_t gpiote_chan_mask;
+/* Reserve channels below the base index */
+#define GPIOTE_CH_BASE CONFIG_GPIO_NRF5_GPIOTE_CHAN_BASE
+static u32_t gpiote_chan_mask = BIT(GPIOTE_CH_BASE) - 1;
```

Comment:
You can use the `BIT_MASK` macro for this.

Reply:
will do